src/Stormpath/Http/HttpClientRequestExecutor.php

```diff
@@ -22,17 +22,26 @@
 use Guzzle\Http\Client;
 use Guzzle\Http\Message\RequestInterface;
 use Stormpath\ApiKey;
+use Stormpath\Http\Authc\RequestSigner;
 use Stormpath\Http\Authc\Sauthc1Signer;
+use Stormpath\Stormpath;
 
 class HttpClientRequestExecutor implements RequestExecutor
 {
+    private $authenticationScheme = Stormpath::SAUTHC1_AUTHENTICATION_SCHEME;
```

Comment:
I guess not... I didn't use it, but then decided to put it in there.  Ill pull it back out.